src/client/auth/auth.proto

```diff
@@ -18,8 +18,8 @@ import "google/protobuf/timestamp.proto";
  * ListAdmins) will have their own conventions
  *
  * The current user formats are:
- * 1) GitHub usernames:
- *      "github:MyGitHubUsername"
+ * 1) Users synced from an identity provider:
+ *      "<idp>:username"
```

Comment:
I think long-term we won't need to support multiple IDPs in pachd at all, so we may even be able to simplify this to just "user:<username>".